examples/tinygo-outbound-pg/README.md

```diff
@@ -0,0 +1,18 @@
+# Spin component with outbound postgres in TinyGo
```

Comment:
Would you mind adding instructions for using `wit-bindgen` 0.3.0 to re-generate the bindings if necessary (e.g. because we add variant cases, new functions, etc. to outbound-pg.wit)?

Reply:
@dicej I added instructions to the readme in the `sdk/go` folder. Let me know if that is too hacky to include :)